consensus/src/main/java/org/apache/iotdb/consensus/ratis/SnapshotStorage.java

```diff
@@ -135,6 +137,28 @@ public SnapshotInfo getLatestSnapshot() {
         fileInfos, snapshotTermIndex.getTerm(), snapshotTermIndex.getIndex());
   }
 
+  /*
+  Snapshot cache will be updated upon:
+  1. initialize, when RaftServer first starts
+  2. reinitialize, when RaftServer resumes from pause. Leader may install a new snapshot during pause.
+  3. takeSnapshot, when RaftServer takes a new snapshot
+  4. loadSnapshot, when RaftServer is asked to load a snapshot
```

Comment:
update the java doc, only need to update cache at the last step of `takeSnapshot` and first step of `loadSnapshot`. `loadSnapshot` will be called by `initialize` and `reinitialize`.

Reply:
updated